schema/types/isolator.go

```diff
@@ -24,6 +24,7 @@ var (
 	isolatorMap map[ACIdentifier]IsolatorValueConstructor
 
 	ErrIncompatibleIsolator = errors.New("isolators set contains incompatible types")
+	ErrInvalidIsolator      = errors.New("invalid isolator")
```

Comment:
Yup, the one above too? In general I haven't seen lot of docstring around error constants, but I will happy to have more.